packages/lit-dev-content/samples/examples/react-basics/index.html

```diff
@@ -0,0 +1,9 @@
+<!doctype html>
+<head>
+  <script src="https://unpkg.com/react@18/umd/react.development.js"></script>
+  <script src="https://unpkg.com/react-dom@18/umd/react-dom.development.js"></script>
+  <script type="module" src="./index.js"></script>
+</head>
+<body>
+  <section></section>
```

Comment:
I would probably do `<div id="app">` as that's very common in React demos. `<section>` might imply the app is a section or something, but regardless, not critical to the example.

Reply:
eh augustine said the same thing in personally idc, let's just change em

Reply:
done!